prophyc/parsers/sack.py

```diff
@@ -169,12 +185,59 @@ def check():
             return SackParserStatus("sack input requires libclang and it's not installed")
         return SackParserStatus()
 
-    def __init__(self, include_dirs=[], warn=None):
+    def __init__(self, include_dirs=[], warn=None, supple_nodes=[]):
         self.include_dirs = include_dirs
         self.warn = warn
+        self.supple_nodes = supple_nodes
+        if supple_nodes:
+            self.prepare_clang_fakery(supple_nodes)
+
+    def prepare_clang_fakery(self, supple_nodes):
```

Comment:
I would really opt for using cursor.extent! This code seems complex and can be easily omitted if type name (alone) is known. We already have all isar nodes in `model.Include`-s, parsing them again by clang seems superfluous.

By the way - generate tempdir is not cleaned upon process exit (could be done by context manager or atexit). This way /tmp gets polluted with no automatic cleaning mechanism in place.